_includes/about.html

```diff
@@ -3,7 +3,7 @@
         <div class="row">
             <div class="col-lg-8 col-lg-offset-2">
                 <h2>About Konstructs</h2>
-                <p>This is a Infiniminer/Minecraft inspired multiplayer
+                <p>This is a modded Minecraft/Minetest inspired multiplayer
```

Comment:
mod, modification and modified minecraft is usaly called modded minecraft. A common lingo.